File: frontend/easypark_mobile/lib/models/city_coordinate.dart
class CityCoordinate {
  final String city;
  final double latitude;
  final double longitude;

  CityCoordinate({
    required this.city,
    required this.latitude,
    required this.longitude,
  });

  factory CityCoordinate.fromJson(Map<String, dynamic> json) {
    return CityCoordinate(
      city: (json['city'] as String?) ?? '',
      latitude: (json['latitude'] as num).toDouble(),
      longitude: (json['longitude'] as num).toDouble(),
    );
  }
}

File: backend/EasyPark.Services/Database/CityCoordinate.cs
namespace EasyPark.Services.Database
{
    public class CityCoordinate
    {
        public int Id { get; set; }
        public string City { get; set; } = null!;
        public decimal Latitude { get; set; }
        public decimal Longitude { get; set; }
    }
}

